core/src/replay_stage.rs

```diff
@@ -1831,7 +1831,7 @@ impl ReplayStage {
                 {
                     assert_eq!(prev_hash, duplicate_confirmed_hash);
                     // Already processed this signal
```

Comment:
nit: maybe change the comment here to say why we use continue instead of return?